front/src/routes/integration/all/philips-hue/device-page/FoundDevices.jsx

```diff
@@ -74,8 +73,10 @@ const FoundDevices = ({ children, ...props }) => (
                   </div>
                 </div>
               ))}
-            {props.philipsHueDevices && props.philipsHueDevices.length === 0 && (
-              <Text id="integration.philipsHue.device.noDevices" />
+            {(!props.philipsHueDevices || props.philipsHueDevices.length === 0) && (
```

Comment:
Philips hue empty state is added 2 times ? 

<img width="928" alt="Screenshot 2022-04-01 at 15 36 15" src="https://user-images.githubusercontent.com/7365207/161285223-27b2f15d-5170-403f-ba32-de86c1c9bd67.png">